### ISSUE TITLE
[Request] Add completionMaterial to settings, when completionCustomModelData is enabled

### ISSUE THREAD

**Issue by PKBCRAFT** [GC-spigot/battle-pass#449]
### Describe the bug

The line with completionCustomModelData is not working, I am adding a number for the model data and it doesn't work at all, the icons from the daily quests don't change.
![image](https://github.com/GC-spigot/battle-pass/assets/147991120/ecda9b2e-4001-46d2-9ded-d7b3192e5d19)
I tried adding the lines like:
material: paper
material : 'paper:0:

None works.
Running version BattlePass-4.0.16

### How to reproduce

1. Go to Battlepass config folder
2. Open settings.yml
3. Find line 85 or where it says completionCustomModelData:
4. Add a number for your custom model data
5. Join your server
6. Complete a daily quest
7. Use the command /bpadmin reload
8. Open Battlepass menu, go to quests > daily quests
9. See that the icon for completed daily quests won't work

**Comment by Timoti11** [comment]
+, me too

**Comment by KPGTB** [comment]
Hey! I can't reproduce that problem. Make sure that your model in resource pack is configured correctly. Also please notice, that icon of quest won't be changed on complete, so if icon is a diamond_pickaxe, then your resource pack needs to have custom model data for diamond pickaxe

**Comment by PKBCRAFT** [comment]
Wow, ok so it worked, I tried with a quest which had a book, and I added the material book in the resource pack. However, this is not a great experience at all, most of the server owners use items as a way to represent the quest, if I have 100 quests that means I have to create 100 icons (in the config I mean) for all the items that I use so I can set a custom model data for when the respective quests are completed. Can this be updated to use only one material for all the icons? This would be a great improvement, right now this is just so much extra work for a small feature.
